TombEngine/Game/effects/item_fx.cpp

```diff
@@ -19,14 +20,53 @@ namespace TEN::Effects::Items
 	{
 		item->Effect.Type = EffectType::Fire;
 		item->Effect.Count = timeout;
-		item->Effect.LightColor = Vector3(0.8f, 0.5f, 0.0f);
+		item->Effect.LightColor = Vector3(250, 176, 0);
+		item->Effect.EffectColor1 = Vector3(1.0f, 0.2f, 0.2f);
+		item->Effect.EffectColor2 = Vector3(1.0f, 0.8f, 0.1f);
+	}
+
+	void ItemColorBurn(ItemInfo* item, const ScriptColor& color1, const ScriptColor& color2, int timeout)
```

Comment:
ItemColorBurn and ItemMagicBurn are basically two copypasted functions with just different effect type assigned. You should instead make a single script function for making custom effect with specific colors and effect type.

Reply:
I merged both functions